src/test/java/qna/domain/QuestionTest.java

```diff
@@ -1,6 +1,36 @@
 package qna.domain;
 
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import qna.CannotDeleteException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
 public class QuestionTest {
     public static final Question Q1 = new Question("title1", "contents1").writeBy(UserTest.JAVAJIGI);
     public static final Question Q2 = new Question("title2", "contents2").writeBy(UserTest.SANJIGI);
+
+    @Test
+    @DisplayName("질문 삭제 권한이 없을 경우")
+    void validateOwner() throws CannotDeleteException {
+        assertThatThrownBy(() -> Q1.delete(UserTest.SANJIGI))
+                .isInstanceOf(CannotDeleteException.class)
+                .hasMessageContaining("질문을 삭제할 권한이 없습니다.");
+    }
+
+    @Test
+    @DisplayName("질문 삭제")
+    void deleteQuestion() throws CannotDeleteException {
+        Q1.delete(UserTest.JAVAJIGI);
+        assertThat(Q1.isDeleted()).isEqualTo(true);
```

Comment:
잘 아시고 계시겠지만,
```suggestion
        assertThat(Q1.isDeleted()).isEqualTo(true);
```
이렇게도 할 수 있습니다:)